soc/xtensa/esp32/esp32-mp.c

```diff
@@ -39,9 +40,10 @@ struct cpustart_rec {
 volatile struct cpustart_rec *start_rec;
 static void *appcpu_top;
 static bool cpus_active[CONFIG_MP_MAX_NUM_CPUS];
+extern void z_sched_ipi(void);
```

Comment:
This is defined in ksched.h, FWIW.  But checking the code I don't see it called in this C file (it isn't useful to SOC code), just remove.